.github/workflows/check-author-included.yml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 
+permissions: {}
```

Comment:
```suggestion
permissions: {}  # disables all GitHub permissions for the workflow
```

I only suggest this since it is not obvious what `permissions: {}` does, and it's buried in the [GitHub documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#defining-access-for-the-github_token-permissions).

Reply:
That's a good point!  When I first ran `zizmor` and found this potential vulnerability, I needed to look it up to understand what was going on.   I'll include this comment in `ci.yml` since this is the most frequently read & revised workflow file.